i18n/translations/en/common.json

```diff
@@ -38,6 +38,7 @@
     "number": "Enter a number.",
     "email": "Enter a valid email address.",
     "date": "Enter a valid date in mm/dd/yyyy format.",
```

Comment:
Do we need to modify this part?

Reply:
Not using that on the new input. Left the code for the old input just in case- may circle back and remove completely later.